### PR TITLE
Abstract-interface should have only non-wasm32 dependencies

### PR DESCRIPTION
**Pull request by Buckram123**

### CODE REVIEW

File: framework/packages/standards/dex/Cargo.toml [hunk 1]
```diff
@@ -39,8 +39,6 @@ abstract-adapter = { version = "0.23.0", path = "../../abstract-adapter" }
 
 ibc-chain-registry = { version = "0.25.0", optional = true }
 
-[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-
```
Comment (Buckram123) [inline framework/packages/standards/dex/Cargo.toml:43]: This compile will be 0 seconds on dex standard

File: framework/packages/abstract-interface/Cargo.toml [hunk 2]
```diff
@@ -12,7 +12,7 @@ daemon = ["cw-orch/daemon"]
 integration = []
 interchain = ["dep:cw-orch-interchain", "dep:cw-orch-polytone"]
 
-[dependencies]
```
Comment (CyberHoward) [inline framework/packages/abstract-interface/Cargo.toml:15]: won't this fail to compile without an architecture flag in the library as well? (in lib.rs)

File: framework/packages/standards/dex/Cargo.toml [hunk 3]
```diff
@@ -39,8 +39,6 @@ abstract-adapter = { version = "0.23.0", path = "../../abstract-adapter" }
 
 ibc-chain-registry = { version = "0.25.0", optional = true }
```
Comment (CyberHoward) [inline framework/packages/standards/dex/Cargo.toml:40]: We should be able to remove this dep

File: framework/packages/abstract-interface/Cargo.toml (hunk 2 above)
Comment (Buckram123) [reply framework/packages/abstract-interface/Cargo.toml:15]: > won't this fail to compile without an architecture flag in the library as well? (in lib.rs)

We already have it!
https://github.com/AbstractSDK/abstract/blob/e5f742d505b3934b2c83223b7464980b07fff47d/framework/packages/abstract-interface/src/lib.rs#L1

**Review by CyberHoward — APPROVED** [review]

**Pull request merged**